resources/source-types/apache_common.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: bindplane.observiq.com/v1beta
+kind: SourceType
+metadata:
+  name: apache_common
+  displayName: Apache Common
+  icon: /icons/sources/apache_common.svg
+  description: Log parser for Apache common format
+spec:
+  version: 0.0.1
+  supported_platforms:
+    - linux
+    - windows
+    - macos
+  parameters:
+    - name: file_path
+      label: Access Log File Path(s)
+      description: Path to apache common formatted log file
+      type: strings
+      default: ["/var/log/apache2/access.log"]
+
+    - name: start_at
+      label: Start At
+      description: At startup, where to start reading logs from the file (`beginning` or `end`)
+      type: enum
+      validValues:
+        - beginning
+        - end
+      default: end
+      advancedConfig: true
+
+  logs:
+    receivers: |
+      - plugin/apachecommon:
```

nit: I try to avoid naming receivers because bindplane will already be renaming them to something unique, and it can get pretty long.

```suggestion
      - plugin:
```

---

I'll keep that in mind moving forward and update it here.